nemo_skills/mcp/servers/wikipedia_tool.py

```
@@ -0,0 +1,740 @@
+        "list": "search",
+        "srsearch": query,
+        "srlimit": n,
+        "srprop": "snippet|titlesnippet|sectionsnippet",
+        "format": "json",
+        "formatversion": "2",
+    }
+    headers = {"User-Agent": USER_AGENT, "Accept": "application/json"}
+    async with httpx.AsyncClient(headers=headers) as client:
+        try:
+            data = await _http_get_json(client, ACTION_BASE, params=params)
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)